tests/unit/test_routes.py

```diff
@@ -448,7 +470,7 @@ def test_update_resource(module_client, module_db, fake_auth_from_oc, fake_algol
             languages=["New language"],
             category="New Category",
             url="https://new.url",
-            paid=False,
```

Comment:
The test from line 174 is sufficient, this shouldn't need to change. Can you explain why this is necessary?

Reply:
In that route we do the same lower() call and updating this test should cover that scenario. It's not necessary though, and I don't have an issue removing it

Reply:
Ok I see what you did there. I'd like to make sure both scenarios are covered though. Is there another instance where `update_resource` is called in the tests with `paid=False`? If not, can we add a specific test so that it covers both paths?